app/controllers/application_templates_controller.rb

```diff
@@ -19,6 +19,15 @@ def new
   def show
   end
 
+  def set_active
```

Comment:
I think I'd prefer to accomplish this in one of two ways:

1. Have a controller action which toggles the `active` attribute rather than having to specify it in a parameter - that way, you only need to write the link once, and you can make use of `ActiveRecord::Persistence#toggle`;
2. Instead of writing a controller action just for making DB changes, write a specialized form which just makes a PATCH request. The downside of this is that since the request will go to the `#update` controller action, it will redirect/render the same things the normal editing structure would.

I prefer the first solution.